SyllabusPlusSchedulerService/SchedulerService.cs

```diff
@@ -84,6 +86,26 @@ private void ScheduleCallback(object e)
         }
 
         /// <summary>
+        /// Verify the folder exists before scheduling a recording. If not, put in RR's default folder (just set folder GUID to empty).
+        /// </summary>
+        private ScheduledRecordingResult VerifyFolderAndScheduleRecording(SessionManagementWrapper sessionManagementWrapper, Schedule schedule, RemoteRecorderManagementWrapper remoteRecorderManagementWrapper)
+        {
+            // Verify the folder exists before trying to schedule a recording
+            try
+            {
+                sessionManagementWrapper.GetFolderById(schedule.FolderId);
+            }
+            catch (Exception e)
+            {
+                // If folder with specified ID can't be found, set folder ID to Guid.Empty to create the recording in the Remote Recorder default folder.
+                log.Error(String.Format("Can't find folder with ID {0} in the Panopto database.", schedule.FolderId), e);
+                log.Warn(String.Format("Recording will be scheduled in folder with ID {0}.", this.configSettings.PanoptoDefaultFolder));
+                schedule.FolderId = this.configSettings.PanoptoDefaultFolder;
```

Comment:
If it isn't set or doesn't parse, it gets set to Guid.Empty. I added a function for this in the config helper.

If the Guid is formatted correctly but doesn't correspond to a folder, it would end up in the RR's default folder.